ApplyToBecomeInternal/ApplyToBecomeInternal/wwwroot/src/index.scss

```diff
@@ -15,3 +15,6 @@ for details on configuring this project to bundle and minify static web assets.
 .app-align-right {
   text-align: right;
 }
+.prepare-text-align-right {
+	text-align: right !important;
```

Comment:
I know that the !important declaration should only be used in very specific circumstances.
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity
Are you sure its necessary in this instance?

Reply:
This was added on advice from Ming to resolve the css issue we were having around the right alignment using the govuk css.

Reply:
@mingoland, is this the right place to be using important!  ??

Reply:
Hi @jordan-beard @aishajilani , yes it's basically the same class. I didn't realise that the `app-align-right` exists so I was suggesting that to Aisha only. My fault :) 

In fact, ideally we should use the Design System class `govuk-!-text-align-right` (https://design-system.service.gov.uk/styles/typography/#text-alignment-override-classes) if possible, but for some reasons I think we found that this class is not being included in our build (at least in Aisha's local environment). I wonder if this is a versioning problem about the Govuk front-end library but I will leave it to you.

Reply:
I'm not sure how we import the gov uk design system. Will take a look.

Reply:
I'd suggest to ship it asap and then come back to this one later if possible. If we know that the versioning of govuk DS is a problem then it may deserve us spending a bit more time on it so a long term issue can be fixed. We can still refactor this page and other pages in one go after the import issue is addressed. But of course it's just my suggestion only. It's your call :)